.github/workflows/test-go-cli.yaml

```diff
@@ -5,32 +5,32 @@ on:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Do we need to pin this to this version? Can't we rather use the newer version of Ubuntu seeing one of our projects is already on this new version?

Reply:
Updated to 22.04 in [70b02e8](https://github.com/openhie/package-starter-kit/pull/14/commits/70b02e8ab62dae04a74398dff39d2402b597a37f)